src/component/FetchItems.js

```diff
@@ -10,6 +10,7 @@ const FetchItems = ({ token, setToken, firestore }) => {
   const [empty, setEmpty] = useState(true);
   const concatPath = `/lists/${token}/items`;
   const today = dayjs(new Date());
+  const [filteredInput, setFilteredInput] = useState('');
```

Comment:
This is a non-blocker and fully just a suggestion/comment: but, I might recommend developing a habit of using `null` to represent the absence of something. So, here and when you reset the input (line 122), what I believe we're trying to represent is that there is absolutely nothing in the input field. In that case, I feel like `null` better represents the absence of a value.

Reply:
Thanks for your input Stacie!

I agree that being explicit about the absence of value is a great point, but I think that the advantages of using the same data type from start to finish are worth it.

IMO, I think that if you stick to one data type + avoiding null/undefined, your code will be less error-prone because:

1. You know that you're always dealing with the same type of value (string, array, obj, number, boolean) so you don't have to think about _what would be the value of this when X happens?_
2. If _somehow_ you try to call a string method on `null/undefined` you'll get errors - try out `undefined.toLowerCase()` or `null.toUpperCase()` for example.

Reply:
Lol! After I told y’all this I tweeted about it cause I thought it might be misguided. See twitters response here: https://twitter.com/the_real_stacie/status/1211136984330014720?s=21

Reply:
that’s totally fine Stacie haha!

For anyone reading this, feel tree to take either of the approaches, both have its own pros/cons. :D

Reply:
Here is another interesting write up my good friend Richard just put together for me. Might be insightful for y’all: https://gist.github.com/rtorr/741ee5d3f5b36cba667546b831b15668